tests/suppressionlist_test.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace tool_emailutils;
+
+defined('MOODLE_INTERNAL') || die();
+
+global $CFG;
+require_once($CFG->dirroot . '/admin/tool/emailutils/classes/task/update_suppression_list.php');
```

Comment:
You don't need to include these files, you can autoload `update_supression_list` with `use`:

```php
namespace tool_emailutils;

use tool_emailutils\task\update_supression_list;

/**
 * Test case for suppression list functionality.
....
```

`suppression_list` should be available already as its in the same namespace as the test.